packages/amplify-e2e-core/src/init/deleteProject.ts

```diff
@@ -6,20 +6,12 @@ export const deleteProject = async (cwd: string, profileConfig?: any, usingLates
   const { StackName: stackName, Region: region } = getBackendAmplifyMeta(cwd).providers.awscloudformation;
   await retry(
     () => describeCloudFormationStack(stackName, region, profileConfig),
-    stack => stack.StackStatus.endsWith('_COMPLETE'),
+    stack => stack.StackStatus.endsWith('_COMPLETE') || stack.StackStatus.endsWith('_FAILED'),
```

Comment:
This is a change I'm merging in from the CLI which John released last week. Seems like a sane check to add, but not a huge deal either way since our cleanup script *should* get these cases within 6 hours anyway.